gtk/setup.cfg

```diff
@@ -50,7 +52,7 @@ zip_safe = False
 
 [options.entry_points]
 toga.backends =
-    linux = toga_gtk
+    UnixLike = toga_gtk
```

Comment:
Registering a backend costs nothing; this should 2 independent `linux` and `freebsd` registrations.